docs/root/start/quick-start/configuration-dynamic-control-plane.rst

```diff
@@ -0,0 +1,114 @@
+.. _start_quick_start_dynamic_control_plane:
+
+Configuration: Dynamic from control plane
+=========================================
+
+Setting up Envoy with dynamic configuration is slightly more complex as you must also set up a control plane
+to provide Envoy with its configuration.
+
+There are a number of control planes compatible with Envoy's API such as `Gloo <https://docs.solo.io/gloo/latest/>`_
+or `Istiod <https://istio.io/latest/docs/ops/deployment/architecture/#istiod>`_.
+
+You may also wish to explore implementing your own control plane, in which case the
+`Go Control Plane <https://github.com/envoyproxy/go-control-plane>`_ provides a reference implementation
+that is a good place to start.
+
+At a minimum, you will need to start Envoy configured with the following sections:
+
+- :ref:`node <start_quick_start_dynamic_node>` to uniquely identify the proxy node.
+- :ref:`dynamic_resources <start_quick_start_dynamic_dynamic_resources>` to tell Envoy which configurations should be updated dynamically
+- :ref:`static_resources <start_quick_start_dynamic_static_resources>` to specify where Envoy should retrieve its configuration from.
+- :ref:`layered_runtime <start_quick_start_dynamic_layered_runtime>` to persist dynamically-provided configurations.
```

Comment:
This isn't actually required. Would it be simpler to just skip RTDS in the overview? WDYT?

Reply:
in the sandbox it appeared to be required to make it work

im pro removing if its not absolutely necessary, but i would like to see it work - ill check this again

Reply:
i checked again, and it wasnt necessary, so i removed from both